### PR TITLE
Jm node express example

### PR DESCRIPTION
**Pull request by JacquiManzi**

### CODE REVIEW

File: node-express-react/node-express/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,40 @@
+root = true
```
Comment (mmcc) [inline node-express-react/node-express/.editorconfig]: Whoa, I'm surprised I haven't seen this config before. Should we put this in root of the whole project so it's shared between all the examples?

Comment (JacquiManzi) [reply node-express-react/node-express/.editorconfig]: mm, good questio. I would say no, it no longer makes sense to do this.

**Comment by dylanjha** [comment]
Closing this b/c it's > 2 years old, although I do like this example.

Would love if someone could circle back and get it wrapped up at some point!

**Comment by mmcc** [comment]
Ah shoot...there's definitely a branch somewhere where we took time starting to transition this to some of our modern features. Big things were stuff like direct uploads...we should really try and bring this one in, it's a :fire: example and it's actually designed and such.

**Pull request closed without merging**